packages/components/pie-button/src/defs.ts

```diff
@@ -17,36 +17,36 @@ export interface ButtonProps {
     /**
      * What size the button should be.
      */
-    size: typeof sizes[number];
+    size?: typeof sizes[number];
```

Comment:
Will we need to make similar changes across all other components?

Reply:
Yes, and a ticket for that is ready for refinement (DSW-1577)